custom/icds_reports/static/js/directives/awc-reports/awc-reports.directive.js

```diff
@@ -2023,7 +2023,7 @@ function AwcReportsController($scope, $http, $location, $routeParams, $log, DTOp
         chart: {
             type: 'multiBarChart',
             height: isMobile ? 350 : 450,
-            width: isMobile ? 600 : 1100,
```

Comment:
does this have any negative UI implications on web? curious why this limit was here in the first place.

Reply:
Keeping fixed width makes page horizontally scrollable on smaller views like this. 
![Screenshot from 2020-01-31 18-14-45](https://user-images.githubusercontent.com/56248457/73540387-c9657580-4455-11ea-869f-6406dbaa085b.png)
Now it would take the container width.

Reply:
gotcha. that seems like an improvement to me. trying to think of any reason it's not always better (maybe something with dense charts on small screens?). or maybe it's just a relic no one ever bothered thinking about till now.